bindings/wasm/bindings.js

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 var _ManifoldInitialized = false;
-Module.setup = function () {
+Module.setup = function() {
```

Comment:
Ignore all these formatting changes; apparently I didn't have my VSCode set up to auto-format JS files. We should really add that check to our CI (I think we only check C files currently).

Reply:
will get back into it this weekend and take a shot at integrating JS